src/containers/org/components/PermissionsPanel.js

```diff
@@ -200,7 +287,29 @@ const PermissionsPanel = ({
         {`These are the properties that are assigned the ${_lowerCase(permissionType)} permission.`}
       </Typography>
       <Divider isVisible={false} margin={24} />
-      <PropertyTypesCard>
+      <PermissionCard>
+        <PermissionCardSegment>
+          <Typography variant="body1">All properties</Typography>
+          <IconButton onClick={togglePermissionAssignmentAll}>
+            <FontAwesomeIcon
+                color={isPermissionAssignedToAll ? PURPLE.P300 : NEUTRAL.N500}
+                fixedWidth
+                icon={faToggleOn}
+                transform={{ rotate: isPermissionAssignedToAll ? 0 : 180 }}
+                size="lg" />
+          </IconButton>
```

Comment:
please add the aria-label attribute to buttons that don't contain text. The label should clearly describe the destination, purpose, function, or action for screen reader users.